content/en/news/releases/1.17.x/announcing-1.17/_index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Announcing Istio release-1.17
+linktitle: 1.17
+subtitle: Major Update
+description: Istio 1.17 release annoucement.
+publishdate: 2023-02-14
+release: 1.17.0
+aliases:
+    - /news/announcing-1.17
+    - /news/announcing-1.17.0
+---
+
+We are pleased to announce the release of Istio 1.17. This is the first Istio release of 2023. We would like to thank the entire Istio community for helping get the 1.17.0 release published. We would like to thank the Release Managers for this release - Mariam John from IBM and Paul Merrison from Tetrate. The release managers would specially like to thank the Test & Release WG lead Eric Van Norman (IBM) for his help and guidance throughout the release cycle. We would also like to thank the maintainers of the Istio work groups and the broader Istio community for helping us throughout the release process with timely feedback, reviews, community testing and for all your support to help ensure we release on time and with a focus on quality.
+
+{{< relnote >}}
+
+{{< tip >}}
+Istio 1.17.0 is officially supported on Kubernetes versions `1.23` to `1.26`.
+{{< /tip >}}
+
+## What's new
+
+Here are some of the highlights of the release:
```

Comment:
If we can get https://github.com/istio/istio/pull/43285 merged, then we can add the following:

```suggestion
Here are some of the highlights of the release:

### Upgraded support for the Kubernetes Gateway API

Istio's implementation of the [Gateway API](https://gateway-api.sigs.k8s.io/) has been moved to, and is now fully compliant with, the latest version of the API (0.6.1).
```

Reply:
Done

Reply:
Hmm, I don't see this it in the preview?

btw, I would suggest adding it close to the top, maybe third, after the two beta promotions?

Reply:
Did that and just pushed the commit I was working on. I have been trying to fix a trailing space in this file that the linter is complaining about since morning. That is what took me so long to push the changes (and I am still trying to figure out where that trailing space is :))

Reply:
Hmm, the lines it's complaining about don't seem to have any trailing spaces. Maybe there's some control-character on the line. Did you try deleting lines 26, 42, and 63 and see what happens?

Reply:
It's the line after what's in the log that has extra spaces.

Reply:
I fixed those listing issues. Yesterday the whole day I tried finding what it was complaining about and finally at night I figured out what it was. The real trailing spaces where not on the lines that linter pointed out (26, 42, 63). couple of lines after each of those lines had an extra space and all along I was looking at lines 26, 42 and 63. There are just some days like that.